inc/config.php

```diff
@@ -45,6 +45,25 @@
 Session::setPath();
 Session::start();
 
+//init cache
+try {
+   $cache_conf = GLPI_CACHE_CONFIG;
+   if (!isset($cache_conf['adapter']['options'])
+       || !isset($cache_conf['adapter']['options']['namespace'])) {
+      $cache_conf['options']['namespace'] = 'glpicache';
+   }
+   if (!defined('TU_USER') || defined('CACHED_TESTS')) {
+      $GLPI_CACHE = Zend\Cache\StorageFactory::factory($cache_conf);
+   } else {
+      $GLPI_CACHE = false;
```

Comment:
probably more robust to move default value above (before try), for now there's still run case without initialized value.

Reply:
Done